coralproject/talk
# Verify Email upon signup

## What does this PR do?

- adds a setting in the admin to enable email verification for new users
- email is sent to new users upon sign up
- clicking the link will confirm your account
- if the user got distracted upon signing up the first time, or lost the email, they can request a new one after trying to sign in with the right credentials

## How do I test this PR?

- turn on Require Email Confirmation in the admin
- create a new user
- click the confirm link in your email
- if you try to request an email again if you're already verified, it should bounce you
- try to log in again. it should work.
